### PR TITLE
Added Panache-Flyway example

### PR DESCRIPTION
**Pull request by Sgitario**
Added new module to cover Hibernate ORM and Flyway modules.
Note that my idea is to cover more databases in the future.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,6 +18,8 @@
       <module>007-openshift-yaml-health-probes</module>
       <module>008-infinispan-getting-started</module>
       <module>009-quarkus-infinispan-grpc-kafka</module>
+      <module>010-quarkus-opentracing-reactive-grpc</module>
```
Comment (Sgitario) [inline pom.xml:21]: This module was missing in the parent.

File: 011-quarkus-panache-rest-flyway/src/main/java/io/quarkus/qe/ApplicationEntity.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.validation.constraints.NotEmpty;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+
+@Entity(name = "application")
+public class ApplicationEntity extends PanacheEntity {
+    @NotEmpty
```
Comment (Sgitario) [inline 011-quarkus-panache-rest-flyway/src/main/java/io/quarkus/qe/ApplicationEntity.java:11]: The @NotEmpty annotation is not working. I reported this issue to make the extensions worked together: 
https://github.com/quarkusio/quarkus/issues/13307

**Comment by Sgitario** [comment]
I ran this new module in native mode and works fine.

File: 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/containers/PostgreSqlDatabaseTestResource.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    private static final String DEFAULT_DATABASE = "db";
+    private static final String DEFAULT_SCHEMA = "test";
+    private static final String DEFAULT_USER = "sarah";
+    private static final String DEFAULT_PASSWORD = "connor";
+    private static final String POSTGRESQL = "postgresql";
+
+    private GenericContainer<?> container;
+
+    @Override
+    public Map<String, String> start() {
```
Comment (rsvoboda) [inline 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/containers/PostgreSqlDatabaseTestResource.java:30]: There is https://mvnrepository.com/artifact/org.testcontainers/postgresql/1.15.0

GenericContainer is not necessary to be used.

File: 011-quarkus-panache-rest-flyway/src/main/java/io/quarkus/qe/ApplicationEntity.java (hunk 2 above)
Comment (rsvoboda) [reply 011-quarkus-panache-rest-flyway/src/main/java/io/quarkus/qe/ApplicationEntity.java:11]: The discussion on the issue is not indicating that `@NotEmpty` is not working. Can you please update the issue with more details? Or update the disabled test.

**Review by rsvoboda — CHANGES_REQUESTED** [review]
Changes requested, please also rebase against the master to pick GH actions updates

File: 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/containers/PostgreSqlDatabaseTestResource.java (hunk 3 above)
Comment (Sgitario) [reply 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/containers/PostgreSqlDatabaseTestResource.java:30]: Changed

File: 011-quarkus-panache-rest-flyway/src/main/java/io/quarkus/qe/ApplicationEntity.java (hunk 2 above)
Comment (Sgitario) [reply 011-quarkus-panache-rest-flyway/src/main/java/io/quarkus/qe/ApplicationEntity.java:11]: I've updated the disabled test to provide more details about why is failing. 
Let me know if it looks ok now.
Thanks.

File: 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/containers/PostgreSqlDatabaseTestResource.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    private static final String QUARKUS_DB_USER = "quarkus.datasource.username";
+    private static final String QUARKUS_DB_PASSWORD = "quarkus.datasource.password";
+    private static final String DEFAULT_SCHEMA = "test";
+    private static final String POSTGRESQL = "postgresql";
+
+    private PostgreSQLContainer<?> container;
+
+    @Override
+    public Map<String, String> start() {
+        container = new PostgreSQLContainer<>("postgres:10.5");
```
Comment (rsvoboda) [inline 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/containers/PostgreSqlDatabaseTestResource.java:28]: We should use newer version, 13.1 or 12.5 ... this is my last comment to this :)

Comment (Sgitario) [reply 011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/containers/PostgreSqlDatabaseTestResource.java]: Changes. Thanks!

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**